source/openapi3/split.yaml

```
@@ -393,14 +387,12 @@ info:
     ## Making payments
 
     In order to payout funds, you'll be looking to use the
```

Are we still using the term _payouts_? Or can we simplify this further, eg:

```suggestion
    To make payouts, use the
```